### ISSUE TITLE
How to compare GANs without a test set?

### ISSUE THREAD

**Issue by benstaf** [jiwoongim/GRAN#3]
I have a question about your paper, about the test set x_{test}: suppose I have an unlabeled training set with only one class (only pictures of tables, say). I use 2 GANs to generate more pictures of tables. How can I use your metric to compare them?
